packages/grafana-data/src/types/vector.ts

```diff
@@ -1,3 +1,56 @@
+declare global {
+  interface Array<T> {
+    /** @deprecated this only exists to help migrate Vector to Array */
+    get(idx: number): T;
+    /** @deprecated this only exists to help migrate Vector to Array */
+    set(idx: number, value: T): void;
+    /** @deprecated this only exists to help migrate Vector to Array */
+    add(value: T): void;
+    /** @deprecated this only exists to help migrate Vector to Array */
+    toArray(): T[];
+  }
```

Comment:
Is this extending the native Array interface, or replacing it? I guess TS would yell if it replaced it...